src/app/components/DateFormatter/index.tsx

```diff
@@ -9,14 +9,20 @@ interface Props {
   date: Date | number
 }
 
+// Workaround - https://github.com/parcel-bundler/parcel/issues/4458
+const getTimeZone = () => {
+  if (process && process.env && process.env.NODE_ENV === 'test') return 'UTC'
+  else return undefined
+}
+
 const dateFormat = new Intl.DateTimeFormat(undefined, {
   year: 'numeric',
   month: 'numeric',
   day: 'numeric',
   hour: 'numeric',
   minute: 'numeric',
   second: 'numeric',
-  timeZone: process?.env?.NODE_ENV === 'test' ? 'UTC' : undefined,
+  timeZone: getTimeZone(),
```

Comment:
This is really strange. I can build:
- your version
- `timeZone: process?.env?.NODE_ENV === 'test' ? 'UTC' : undefined,` iff I keep getTimeZone defined above
- `timeZone: globalThis?.env?.NODE_ENV === 'test' ? 'UTC' : undefined,`

Reply:
I think we can just use `timeZone: process.env.NODE_ENV === 'test' ? 'UTC' : undefined`

Reply:
Thank you for taking a look @lukaw3d !

It has to do with 

https://github.com/parcel-bundler/parcel/issues/7101

https://github.com/parcel-bundler/parcel/issues/4458